model-optimizer/extensions/front/LayerNorm.py

```diff
@@ -0,0 +1,57 @@
+"""
+ Copyright (C) 2017-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph
+from extensions.ops.mvn import MVN
+
+
+class LayerNorm(FrontReplacementSubgraph):
+    enabled = True
+    graph_condition = [lambda graph: graph.graph['cmd_params'].generate_experimental_IR_V10]
+
+    def pattern(self):
+        log.info('Enabled LayerNorm replacement')
+        return dict(
+            nodes=[
+                ('pool0',  dict(op='ReduceMean')),
+                ('pool1',  dict(op='ReduceMean')),
```

Comment:
Please, check that the ReduceMean is performed over spatial dimensions only. If it is over feature dimension also then the `across_channels` must be set to 1.